tiktok.cs

```diff
@@ -5,394 +5,105 @@ namespace First_project
     class Program
     {
             
-        static void XTurn(out int xGuessB, out int yGuessB)

```

Comment:
Here you had XTurn and YTurn doing the same thing. You want to avoid repeated code at all costs.